provider/pkg/helm/tool.go

```diff
@@ -130,17 +131,9 @@ func (cmd *TemplateOrInstallCommand) addInstallFlags() {
 	client.SubNotes = false
 	client.Labels = nil
 	client.EnableDNS = false
-	cmd.addValueOptionsFlags()
 	cmd.addChartPathOptionsFlags()
 }
 
-func (cmd *TemplateOrInstallCommand) addValueOptionsFlags() {
```

Comment:
I would still keep `addValueOptionsFlags` to set defaults on `Values` and to further resemble:
https://github.com/helm/helm/blob/14d0c13e9eefff5b4a1b511cf50643529692ec94/cmd/helm/flags.go#L45-L51